src/browser/provider/index.js

```diff
@@ -195,6 +195,14 @@ export default class BrowserProvider {
     }
 
     async closeBrowser (browserId) {
+        var isLocalBrowser    = await this.plugin.isLocalBrowser(browserId);
+        var supportedFeatures = await this.plugin.hasCustomActionForBrowser(browserId);
```

Comment:
`this.plugin.hasCustomActionForBrowser(browserId)` - bad naming convention.

`has` prefix describe that method returns `true` of `false`.

I propose the following variant:
```js
var browserCustomActions = await this.plugin.getCustomActionsForBrowser(browserId)
...
if (isLocalBrowser && !browserCustomActions.closeBrowser) {
```

/cc @AlexanderMoskovkin

Reply:
I agree with this point but it doesn't relate to this PR. And this plugin is already a part of the plugin-host API. If we change it it'll be a breaking change. I think we shouldn't change it in the context of this PR.

Reply:
I've created the separate issue - https://github.com/DevExpress/testcafe/issues/1784